index.html

```diff
@@ -68,6 +68,7 @@
 <body>
 	<form method="post">
 		<input name="refresh" type="button" onclick="refresh_map('all')" value="Refresh"/>
```

Comment:
This 'Refresh' button doesnt work after you add a "Universitas di Flores" button.

The user clicks "Universitas di Flores", then clicks "Refresh", and the result is no longer correct.

UI improvement is needed to support multiple queries properly.